src/screens/Ranking/GroupRanking/GroupRanking.jsx

```diff
@@ -51,6 +59,17 @@ const GroupRanking = ({ name, members }) => {
                   <TableCell padding="none">
                     {(user.score || 0).toLocaleString()} points
                   </TableCell>
+                  <TableCell padding="default">
+                    {team.elimination ? (
+                      <Flag country={team.code} className="bet-winner-beaten" />
+                    ) : (
+                      <img
+                        src={imgUrl().default}
+                        className="bet-winner-unknown"
+                        alt="unknown team"
+                      ></img>
```

Comment:
Pas d'enfants => Balise auto-fermante
```suggestion
                      />
```